drivers/src/soc_ifc.rs

```diff
@@ -285,7 +285,7 @@ impl SocIfc {
         self.soc_ifc
             .regs_mut()
             .cptra_flow_status()
-            .write(|w| w.ready_for_runtime(true));
+            .modify(|w| w.ready_for_runtime(true));
```

Comment:
Just for my education, what's the difference?

Reply:
.modify() is a ready-modify-write that only sets the ready_for_runtime field. .write() does a write that sets all the other fields to the reset value.

Reply:
See docs at https://github.com/chipsalliance/caliptra-sw/blob/955115deef9fd788b78127a5cc06dc5e8eb69aa4/ureg/src/lib.rs#L506

and

https://github.com/chipsalliance/caliptra-sw/blob/955115deef9fd788b78127a5cc06dc5e8eb69aa4/ureg/src/lib.rs#L588

Reply:
Makes sense, thanks :)